db/people.yml

```diff
@@ -5582,6 +5582,11 @@ zthatcher:
       end_month: 3
   email: zt2275@columbia.edu
   employment:
+    - begin_year: 2022
+      begin_month: 3
+      location: Remote
```

Comment:
is location required?  If not, Remote seems pretty arbitrary unless it is actually a place....  Is it `New York, NY`?